osf/models/analytics.py

```diff
@@ -102,8 +124,7 @@ def update_counter(cls, page, node_info):
                     # increment the number of unique visitors for today
                     model_instance.date[date_string]['unique'] += 1
                 else:
-                    # set the number of unique visitors to 1
-                    model_instance.date[date_string] = dict(unique=1)
+                    model_instance.date[date_string] = dict(unique=0)
```

Comment:
Why was this changed?

Reply:
This was a pre-exsisting bug where the first unique user got double counted [here](https://github.com/CenterForOpenScience/osf.io/pull/8478/files/6eb96f148c82c8c8095dbfbd4a7b17fd7c178fe3#diff-4519792f13c44cf51a8eec0288fa674aR149) if IIRC.

Reply:
The code you linked to is modifying `model_instance.unique`, not `model_instance.date`, so I'm unsure what you mean by the double-counting. Can you clarify?